eks/README.md

```diff
@@ -44,6 +44,8 @@ configure the CircleCI Server application:
      This key will be used in the `bastion_key` terraform variable. 
 5. Navigate to `./eks`
 6. Modify the `terraform.tfvars.template` file and save as `terraform.tfvars`
+1. Modify the `remote_state.tf.template` file using the same values from
```

Comment:
I know that MD renders the numbers in their correct order, but can you nonetheless fix the numbering?

Reply:
done